docs/Changelog.md

```diff
@@ -14801,6 +14801,8 @@ This version of the operator has been available since version 12 of the default
 #### Attributes
 
 <dl>
+<dt><tt>ignore_index</tt> : int (default is -100)</dt>
```

Comment:
Where does this default come from? If we don't have a good default value, maybe we can make this attribute non-optional?

Reply:
It is coming from PyTorch but I can make it non-optional.